### PR TITLE
Feature ROBOTIQ grippers

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Add three ROBOTIQ grippers (used by TIAGo robot):
* [x] 2F-85 parallel grippers
* [x] 2F-140 parallel grippers
* [x] EPick vaccum gripper

Tasks:
  * [x] PROTO files
  * [x] TIAGo socket PROTO for ROBOTIQ grippers
  * [x] icons
  * [x] documentation (in the PROTO file)
  * [x] add robot window support for VacuumDevice
  * [x] sample simulation using parallel grippers

### CODE REVIEW

File: resources/projects/libraries/generic_robot_window/generic.c [hunk 1]
```diff
@@ -46,7 +47,7 @@ static int time_step = 0;
 static bool is_hidden = false;
 static double refresh_rate = 0.032;  // s
 static double last_update_time = 0;  // s
-static bool isDeviceTypeControlEnabled[WB_NODE_TOUCH_SENSOR - WB_NODE_ACCELEROMETER];
+static bool isDeviceTypeControlEnabled[WB_NODE_VACUUM_GRIPPER - WB_NODE_ACCELEROMETER + 1];
```
Comment (BenjaminDeleze) [inline resources/projects/libraries/generic_robot_window/generic.c:50]: Was it wrong before?

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you, I have some remarks:

- I think that the motors of the grippers should be coupled
- When replacing the `TiagoGripper` by a `TiagoRobotiqGripper` in tiago.wbt, I get the following error:
```
WARNING: Tiago "TIAGo" > TiagoRobotiqGripper "frontgripper_coupler_link" > Solid "frontgripper_coupler_link" > Robotiq2f85Gripper "front" > HingeJoint "left inner knuckle" > Solid "left inner knuckle" > HingeJoint  > SolidReference : SolidReference has an invalid 'left inner finger' name or refers to its closest upper solid, which is prohibited.
WARNING: Tiago "TIAGo" > TiagoRobotiqGripper "frontgripper_coupler_link" > Solid "frontgripper_coupler_link" > Robotiq2f85Gripper "front" > HingeJoint "left inner knuckle" > Solid "left inner knuckle" > HingeJoint  > SolidReference : SolidReference has an invalid 'left inner finger' name or refers to its closest upper solid, which is prohibited.
WARNING: Tiago "TIAGo" > TiagoRobotiqGripper "frontgripper_coupler_link" > Solid "frontgripper_coupler_link" > Robotiq2f85Gripper "front" > HingeJoint "right inner knuckle" > Solid "right inner knuckle" > HingeJoint  > SolidReference : SolidReference has an invalid 'right inner finger' name or refers to its closest upper solid, which is prohibited.
WARNING: Tiago "TIAGo" > TiagoRobotiqGripper "frontgripper_coupler_link" > Solid "frontgripper_coupler_link" > Robotiq2f85Gripper "front" > HingeJoint "right inner knuckle" > Solid "right inner knuckle" > HingeJoint  > SolidReference : SolidReference has an invalid 'right inner finger' name or refers to its closest upper solid, which is prohibited.
```
- Both grippers do some strange mouvements:

https://github.com/cyberbotics/webots/assets/25938827/8499a87e-5757-4a0d-975a-f8f2f17c0f23

Once this PR is merge we should add the new protos on webots.cloud

File: resources/projects/libraries/generic_robot_window/generic.c (hunk 1 above)
Comment (stefaniapedrazzi) [reply resources/projects/libraries/generic_robot_window/generic.c:50]: Yes, when accessing the `isDeviceTypeControlEnabled` value for `WB_NODE_TOUCH_SENSOR` type, the index was out of bounds: size ``WB_NODE_TOUCH_SENSOR - WB_NODE_ACCELEROMETER`` and accessing index ``WB_NODE_TOUCH_SENSOR - WB_NODE_ACCELEROMETER``.

**Comment by stefaniapedrazzi** [comment]
I coupled the motors and fixed the initial movement of the grippers due to the `position matching the `minStop` value.

> When replacing the TiagoGripper by a TiagoRobotiqGripper in tiago.wbt, I get the following error:

Yes, there are some issues in this case.
However it seems a Webots problem and not a problem with the models, because if you save and revert no error is displayed and the gripper works as expected. I will investigate in a separate PR.

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Pull request merged**